tracer/test/test-applications/integrations/LogsInjection.Log4Net/Properties/launchSettings.json

```diff
@@ -3,7 +3,18 @@
     "LogsInjection.Log4Net": {
       "commandName": "Project",
       "environmentVariables": {
-        "DD_LOGS_INJECTION": "true"
+        "COR_ENABLE_PROFILING": "1",
+        "COR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
+        "COR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\Datadog.Trace.ClrProfiler.Native.dll",
+
+        "CORECLR_ENABLE_PROFILING": "1",
+        "CORECLR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
+        "CORECLR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\Datadog.Trace.ClrProfiler.Native.dll",
+
+        "DD_DOTNET_TRACER_HOME": "$(ProjectDir)$(OutputPath)profiler-lib",
+        "DD_INTEGRATIONS": "$(ProjectDir)$(OutputPath)profiler-lib\\integrations.json",
```

Comment:
If I merge, https://github.com/DataDog/dd-trace-dotnet/pull/1813 before this is merged, don't forget to update the paths

Reply:
Good call, updated in f853879706bc65e9950e52c7e9eefbf84deedbb0